src/main/java/org/opentripplanner/middleware/otp/response/Itinerary.java

```diff
@@ -71,7 +72,8 @@ public class Itinerary implements Cloneable {
     /**
      * How many transfers are part of this itinerary.
      */
-    public Integer numberOfTransfers = 0;
+    @JsonProperty("numberOfTransfers")
```

Comment:
Remove `@JsonProperty` please. The renaming is handled at the GraphQL query level already. Otherwise, itineraries with transfers incorrectly report `transfers = 0`.